src/SelectPicker/SelectPicker.tsx

```diff
@@ -83,6 +83,9 @@ export interface SelectProps<T = ValueType> {
   /** Called when the option is selected */
   onSelect?: (value: any, item: ItemDataType, event: React.SyntheticEvent) => void;
 
+  /** Called when the option is created */
+  onCreate?: (value: any, item: ItemDataType, event: React.SyntheticEvent) => void;
```

Comment:
`onCreate` is a property of InputPicker, so it should not be declared in SelectPicker.